flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/dataformat/BinaryRowTest.java

```diff
@@ -294,6 +294,12 @@ public void anyNullTest() throws IOException {
 			BinaryRowWriter writer = new BinaryRowWriter(row);
 			assertFalse(row.anyNull());
 
+			writer.setNullAt(24);
+			assertTrue(row.anyNull());
+
+			writer.setNullAt(55);
```

Comment:
This is useless, because the previous `setNull` has allowed `anyNull` to return to true.
How about add a test:
```
int numFields = 80;
			for (int i = 0; i < numFields; i++) {
				BinaryRow row = new BinaryRow(80);
				BinaryRowWriter writer = new BinaryRowWriter(row);
				assertFalse(row.anyNull());
				writer.setNullAt(i);
				assertTrue(row.anyNull());
			}
```

Reply:
It's a good advise. I will change it.